models/agent_helpers.go

```diff
@@ -342,6 +342,58 @@ func ChangeAgent(q *reform.Querier, agentID string, params *ChangeCommonAgentPar
 	return row, nil
 }
 
+// PMMAgentsForNode gets pmm-agent for node.
+func PMMAgentsForNode(q *reform.Querier, nodeID string) ([]*Agent, error) {
```

Comment:
https://github.com/percona/pmm-managed/blob/a4b1a0b7ccf26cf4ca678ad103bdce3cd72a497a/models/models.go#L19-L26
Please rename it to `FindPMMAgentsForNode`.

I also think the comment should be expanded: "for node" is too generic.

Lastly, is it the same function as `PMMAgentsForChangedNode` should be (not as it is coded now)? If yes, we should merge them.

And there are _have to be_ tests for this function.

Reply:
> Lastly, is it the same function as PMMAgentsForChangedNode should be (not as it is coded now)? If yes, we should merge them.

I'm not sure what is this function all about. I didn't see it. It's out of the context of this task, isn't it? Also, this function used only in tests. Maybe we can move this in another task?

Reply:
>Also, this function used only in tests.

That's actually a huge problem… k, skip that part, I will take a look

Reply:
>Also, we should rename all old methods like: AgentFindByID, AgentFindAll, AgentAdd*, AgensForNode* and all other stuff, as this all methods breaking the agreement above.

I will do that once this PR is merged.